x-pack/plugins/ingest_manager/public/applications/ingest_manager/sections/agent_config/details_page/components/yaml/index.tsx

```diff
@@ -47,7 +55,17 @@ export const ConfigYamlView = memo<{ config: AgentConfig }>(({ config }) => {
         <EuiCodeBlock language="yaml" isCopyable>
```

Comment:
can be done in a separate PR but it would be nice to have `overflowHeight` prop set, currently the page scrolls very long depending on the config